### DISCUSSION TITLE
Does lighthouse 10.x & 11.x support WCAG 2.1 AA?

### DISCUSSION THREAD

**Discussion by JamsonChan** [GoogleChrome/lighthouse#15692] [Q&A]
Hi, as the title, I only saw this issue https://github.com/GoogleChrome/lighthouse/issues/9507 discuss on this topic, which was 4 years ago. 

However, there is no explicit mention of the relationship between each version and WCAG 2.1 AA on the official Lighthouse documentation.

Therefore, I would like to inquire here if the latest versions, specifically 10.x and 11.x, provide support for WCAG 2.1 AA.

Thank you!

**Comment by adamraine** [comment]
I don't think we can define an explicit relationship with WCAG 2.1 and the previous answer on this topic is still valid. We've added new axe rules since then recently but some of the criteria in WCAG 2.1 [will always need manual verification](https://www.deque.com/blog/support-for-wcag-2-1-in-axe-core/#:~:text=Which%20brings%20us,web%20platform%20APIs.).

**Reply by JamsonChan** [reply]
Okay I got it, thanks🙏🏻
